tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -569,6 +569,23 @@ void runAndVerify(ID3D12Device *D3DDevice, bool VerboseLogging,
                                 ValidationConfig.Type, VerboseLogging));
 }
 
+template <typename IN_TYPE, typename OUT_TYPE>
```

Comment:
You could simplify this to just use <typename T> 
No need to have an in/out type. Not being used.